e2e/utils.go

```diff
@@ -378,7 +378,7 @@ func checkDataPersist(pvcPath, appPath string, f *framework.Framework) error {
 
 	_, stdErr, err := execCommandInPod(f, fmt.Sprintf("echo %s > %s", data, filePath), app.Namespace, &opt)
 	if err != nil {
-		return nil
+		return fmt.Errorf("failed to exec command in pod: %w", err)
```

Comment:
When you notice duplication like the above writing of data to a file, you can always create a helper function that returns a single error instead of `stdErr` and `err`. Not required for this PR, but small things like that are good cleanups too!

Reply:
Yes sure. I will do it another PR.